assets/scss/carousel.scss

```diff
@@ -0,0 +1,27 @@
+@import "_functions";
+@import "_mixins";
+@import "_variables";
+
+.block-lb-carousel {
```

Comment:
@oleh-rymar you have to work with styles of this component in the module file for this component
https://git.drupalcode.org/project/lb_carousel/-/blob/2.0.x/assets/scss/lb-carousel.scss?ref_type=heads

Reply:
https://git.drupalcode.org/project/lb_carousel/-/merge_requests/29/diffs

Reply:
https://www.drupal.org/project/lb_carousel/issues/3438099#comment-15577663 tested and merged.

Reply:
(now with library version bump)